modules/update-mco-process.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// * updating/understanding_updates/how-updates-work.adoc
+
+:_content-type: CONCEPT
+[id="mco-update-process_{context}"]
+= Machine Config Operator node updates
+The Machine Config Operator (MCO) applies a new machine configuration to each control plane and compute node. During this process, the MCO performs the following sequential actions in the cluster:
```

Comment:
MCO will apply different machine configs for control plane nodes and worker nodes.

Reply:
MCO will generated rendered config for all machine-config-pool separately. e.g `rendered-master-xx`, `rendered-worker-xx`, update process is same